src/collective/fingerpointing/upgrades/profiles/1_to_2/registry.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
```

Comment:
we prefer not to add this file, but the new field in the default profile and re-registering from there.